source/user/installation_guide.rst

```diff
@@ -3,58 +3,9 @@ Installation Guide
-installations.
-
-
-Client-Server Installation
---------------------------
-Here is what needs to be done to use this framework.
-
-
-On Laptop/Desktop
-+++++++++++++++++
-+ Fork `PowerSimData <https://github.com/Breakthrough-Energy/PowerSimData>`_ (see our
-  :doc:`git_guide` guide to set this up)
-+ Generate authentication keys (SSH keys) using the ``ssh-keygen`` program
-+ Create a **PowerSimData/powersimdata/utility/.server_user** file enclosing your
-  username on the server if different from your local username
-+ Edit **PowerSimData/powersimdata/utility/server_setup.py** file as follows:
-    + Set ``SERVER_ADDRESS``: your server address
-    + Set ``DATA_ROOT_DIR``: root of the file system
-    + Set ``EXECUTE_DIR``: scenario inputs and outputs will be stored there - note that
-      these files are temporary files and are not necessary to load an already ran
-      scenario
-    + Set ``INPUT_DIR``: all scenario inputs are stored there - after the scenario has
-      finished running and the temporary data are extracted, the **MAT-file** that
-      encloses the grid used for the scenario is stored there along with the **pickle**
-      file enclosing the change table listing the transformation applied to the
-      original grid
-    + Set ``OUTPUT_DIR``: all scenario outputs are stored there after being extracted -
-      the list of output data is given in the :doc:`../powersimdata/index` tutorial
-    + Set ``MODEL_DIR``: this directory encloses the simulation engine
-    + Set ``LOCAL_DIR``: this directory will enclose your local file system
-    + Note that ``BACKUP_DATA_ROOT_DIR`` does not need to be set unless you have a
-      backup storage mounted on you server that you plan to use to transfer scenario
-      data.
-+ Install Python and its dependencies as instructed in our :doc:`python_guide` guide
-+ Fork `PostREISE <https://github.com/Breakthrough-Energy/PostREISE>`_ to later analyze
-  and plot scenario data.
-
-
-On Server
-+++++++++
-+ Copy the generated public key in your **~/.ssh/authorized_keys** file
-+ Install Julia, Python and Gurobi
-+ Create the above mentioned directories, i.e., **DATA_ROOT_DIR**, **EXECUTE_DIR**,
-  **INPUT_DIR**, **OUTPUT_DIR** and **MODEL_DIR**
-+ Create the **ScenarioList.csv** and **ExecuteList.csv** files in **DATA_ROOT_DIR**
-  on the server following templates in **powersimdata/utility/templates/**
-+ Fork `REISE.jl`_ into **MODEL_DIR**
-  and install it following the instructions located in the :doc:`../reisejl_package`
-  README
+There are multiple ways to install this software. For most people, using docker will be the easiest and fastest way to run a simulation. Note that using the Gurobi solver is recommended for this framework, but only certain license types are compatible with the docker installation method. If using Gurobi licenses that are not supported in the docker method, this software can also be installed and run natively with slightly more effort.
```

Comment:
I think it would be good to identify which Gurobi licenses that can be used. To my knowledge for Academia users they would have to go with the Web License Service (https://www.gurobi.com/academia/academic-program-and-licenses/)
For Commercial users WLS, Floating License, Server License or Cloud License works.

Reply:
Makes sense! Do you think it should be in this documentation here or in the `plug` [README](https://github.com/Breakthrough-Energy/plug#prerequisites) (i.e. with a link to `plug` for further details) or both? Either way, this information is missing from our documentation.

Reply:
Can we just point towards the Gurobi website?

I don't think we should say that Gurobi is "recommended", but maybe we say that we use Gurobi internally, and we recommend the use of commercial solvers in general.

Reply:
In that case, I altered the language to be more general and pointed towards `plug` for additional info. I think we should update the `README` there to reflect the above information about recommendations and compatible license types.